jac-uk/admin
# Google Analytics integration with Vue library

## What's included?
closes #2328 
## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to the preview link: https://jac-admin-develop--pr2371-feat-admin-2328-goog-xiberd2m.web.app/
- Visit different pages and different exercises
- Go to the GA portal for Dev environment: https://analytics.google.com/analytics/web/?authuser=0&hl=en#/p436439933/realtime/overview?params=_u..nav%3Dmaui
- Check the `Views by Page title and screen name` card, the URL paths of the pages will appear on the card
<img width="1512" alt="Screenshot 2024-04-16 at 16 04 11" src="https://github.com/jac-uk/admin/assets/156695133/0dcc7f30-a27f-45ac-be89-90e2167c711b">
- The format of path will be across all exercises, ex. `/exercise/:id/dashboard`

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, video demo, notes etc.

## Related permissions
Have permissions been considered for this functionality?
- Can access Google Analytics Portal

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
